AUTHORS.rst

```diff
@@ -64,31 +65,30 @@ Aviv Salem
      PLAIN authentication support
 
 Bug Reports
-------------
-Brian Jackson
-Neil Martinsen-Burrell
-Mark Eichin (eichin _AT_ thok.org)
-Fergal Daly (fergal _AT_ esatclear.ie)
-Bob Yexley (bob _AT_ yexley.net)
-Rhett Garber
-Erik Quaeghebeur
-Benjamin Morrise
-Brandon Rhodes
-Thomi Richards
-Ivan Zderadicka
-Raffael Vogler
-
-External Projects
------------------
+~~~~~~~~~~~
+- Brian Jackson
+- Neil Martinsen-Burrell
+- Mark Eichin (eichin _AT_ thok.org)
+- Fergal Daly (fergal _AT_ esatclear.ie)
+- Bob Yexley (bob _AT_ yexley.net)
+- Rhett Garber
+- Erik Quaeghebeur
+- Benjamin Morrise
+- Brandon Rhodes
+- Thomi Richards
+- Ivan Zderadicka
+- Raffael Vogler
+
+
+Other Contributions
+~~~~~~~~~~~~~~~~~~~
 IMAPClient uses Michael's excellent 'Mock' package.
```

Comment:
I tend to agree. I would also remove the "who did what" sentences in the Code section, which is hard to maintain and just keep the contributor list in alphabetic order, with active maintainers at the top of the list. 
I would also remove the Bug Reports section which is not really "authors" but more thankful users.